backend/routes/users.js

```diff
@@ -1,22 +1,59 @@
-const router = require('express').Router();
-const User = require('../models/user');
+const router = require("express").Router();
+const User = require("../models/user");
 
-router.route('/').get((req, res) => {
-    User.find()     //list of all users in User model
-        .then(users => res.json(users))
-        .catch(err => res.status(400).json('Error: ' + err));
+router.route("/").get((req, res) => {
+  User.find() //list of all users in User model
+    .then((users) => res.json(users))
+    .catch((err) => res.status(400).json("Error: " + err));
 });
 
-router.route('/add').post((req, res) => {
-    const username = req.body.name;
-    const mail = req.body.email;
+function emailRegex(input) {
+  let regex = /[a-zA-Z]+@khealth\.com/i;
+  return regex.test(input);
+}
+
+router.route("/login").post((req, res) => {
+  const mail = req.body.email;
+  const password = req.body.password;
+  User.find({ mail: mail }, async (err, result) => {
+    if (err) {
+      return res.status(500);
+    }
+    console.log(result);
+    if (result.length === 0) {
+      return res.status(400).json({ line: "Email is wrong. Please try again" });
```

Comment:
Better to be consistent with the `return` before `res.status(...`.

I like it like with the return but it should be consistent on all the `res.status(...`.

Reply:
added "return" to all relevant lines.